plugin-gradle/src/test/java/com/diffplug/gradle/spotless/ShellExtensionTest.java

```diff
@@ -24,18 +24,42 @@
 @ShfmtTest
 public class ShellExtensionTest extends GradleIntegrationHarness {
 	@Test
-	void shfmt() throws IOException {
-		setFile("build.gradle").toLines(
+	void shfmtWithEditorconfig() throws IOException {
+		String fileDir = "shell/shfmt/with-config/";
+		setFile(".editorconfig").toResource(fileDir + ".editorconfig");
+		setFile("build.gradle.kts").toLines(
 				"plugins {",
-				"  id 'com.diffplug.spotless'",
+				"  id(\"com.diffplug.spotless\")",
 				"}",
+				"repositories { mavenCentral() }",
 				"spotless {",
 				"  shell {",
 				"    shfmt()",
```

Comment:
If we can add an option to override `.editorconfig` path like `BaseKotlinExtension.KtlintConfig#setEditorConfigPath`?

Reply:
From what I can tell, when configuring ktlint for formatting, you actually provide the .editorconfig path to ktlint and that is an input supported by ktlint. As far as I can tell, there is no way to provide an `.editorconfig` path to `shfmt` 🙁 

With that said, I did request that an input flag be added for situations like this: https://github.com/mvdan/sh/issues/1055